src/main/javascript/src/app.js

```diff
@@ -99,6 +99,41 @@ export class App {
 				moduleId: PLATFORM.moduleName('pages/login/login'),
 				title: "Login",
 			},
+			{
+				show: false,
+				route: '/term/update',
+				name: 'update term',
+				moduleId: PLATFORM.moduleName('pages/term/update/update'),
+				title: "Update Term"
+			},
+			{
+				show: false,
+				route: '/term/create',
+				name: 'create term',
+				moduleId: PLATFORM.moduleName('pages/term/create/create'),
+				title: "Create Term"
+			},
+			{
+				show: false,
+				route: '/user/create',
+				name: 'create user',
+				moduleId: PLATFORM.moduleName('pages/user/create/create'),
+				title: "Create User"
+			},
+			{
+				show: false,
+				route: '/user/update',
+				name: 'update user',
+				moduleId: PLATFORM.moduleName('pages/user/update/update'),
+				title: "Update User"
+			},
+			{
+				show: false,
+				route: '/user/updateInterest',
+				name: 'update user interest',
+				moduleId: PLATFORM.moduleName('pages/user/updateInterest/updateInterest'),
+				title: "Update User Interest"
```

Comment:
I'm gonna try to get it to work this way (still working on it) but i want your PR #118 merged first anyways because it fixed some stuff on the home page that will probably make a conflict somewhere

Reply:
I know this is going to be a little more complicated, so if you want me to show you an example of those composite pages, I'd be happy to. Honestly though, we're running out of time, so I'd be happy as long as I can't go to these pages as a student (or prof?). Even though we're short on time, I think this is still a fast solution because everything pretty well stays the way it is.